src/UI/WB.UI.Headquarters.Core/Resources/Pages.resx

```diff
@@ -736,8 +736,11 @@
   <data name="GlobalSettings_ForbiddenMessage" xml:space="preserve">
     <value>You do not have permission for this action.</value>
   </data>
-  <data name="GlobalSettings_DeleteQuestionnareConfirmation" xml:space="preserve">
-    <value>Are you sure you want to delete questionnaire? All interviews conducted by current questionnaire will be lost.</value>
+  <data name="GlobalSettings_DeleteQuestionnaireConfirmation" xml:space="preserve">
+    <value>Are you sure you want to delete questionnaire? All interviews conducted by current questionnaire, including the interviews collected on the tablets, even those that you don't see on the server currently will be lost.</value>

```

Comment:
you change message message only in this file,
need change message in ru resx and delete it in other resource files